### DISCUSSION TITLE
Let's share our Portfolios! 🌳 (🇫🇷/🇬🇧)

### DISCUSSION THREAD

**Discussion by MadeInPierre** [MadeInPierre/finalynx#75] [Show and tell]
Hi & Bonjour ! :wave: 

<details>
<summary><strong>Texte en Français</strong> :fr: :baguette_bread: </summary>

Finalynx permet d'organiser ses investissements selon ses vrais projets de vie sous forme de structure libre, plutôt que simplement par classe d'actifs ou enveloppes. C'est un projet perso qui prend du temps, au point de m'y attacher un peu :smile:

Ce serait un plaisir de voir le portefeuille structuré d'autres personnes, à la fois pour :
1. Apprécier le résultat de Finalynx obtenu par d'autres personnes,
2. Repérer des améliorations possibles pour Finalynx,
3. Mais aussi et surtout pour partager et s'inspirer de nos manières d'organiser nos finances !

Partagez un screenshot de votre portefeuille ici, avec ou sans explications dans la langue de votre choix :smile: 

```bash
python your_config.py targets         # option 1, si vous avez défini des targets sur chaque noeud
python your_config.py --hide-amounts  # option 2, pour afficher l'arbre complet sans les valeurs d'investissement
python your_config.py                 # option 3, pour partager ouvertement ce que vous possédez !
```

</details>

<details>
<summary><strong>Text in English</strong> :gb: :tea:</summary>

Finalynx allows you to organize your investments according to your real life projects in a flexible structure, rather than simply by asset classes or accounts. It's a personal project that takes some time, to the point where I've become quite attached to it :smile:

It would be a pleasure to see how other people have structured their portfolios, both to:
1. Appreciate the results of Finalynx for other people,
2. Identify possible improvements for Finalynx,
3. But most importantly, to share and draw inspiration from the various ways we organize our finances!

Share a screenshot of your portfolio here, with or without explanations in the language of your choice. :smile:

```bash
python your_config.py targets         # option 1, if you have set targets for each line/folder
python your_config.py --hide-amounts  # option 2, show your full structure without amounts
python your_config.py                 # option 3, openly share whatever you have right now!
```

</details>

**Comment by MadeInPierre** [comment]
:fr: Je commence avec mon portefeuille ! Bon, presque aucun de mes "targets" n'est respecté, faut que je suive ce que Finalynx me recommande :sweat_smile: Je réfléchis encore à quelques derniers ajustements (genre combien de SCPIs et à quels pourcentages) avant d'investir pour de bon. Commentaires bienvenus !

:gb: I'll start with my portfolio! As you can see almost none of my targets are checked, I have to invest in what Finalynx is telling me very soon :sweat_smile: Still thinking about some last adjustments (e.g. how many SCPIs and what percentages) before investing for real. Any comments appreciated!

<img src="https://github.com/MadeInPierre/finalynx/assets/12670280/aac6c891-f9d0-43f4-bdb2-2ccb4e0d3800" width="500"/>

**Comment by nmathey** [comment]
Hello,

Here is mine ;)
I prefer to keep the render simple from Finalynx for now to make macro decision then use either Finary and Moning.co for more specific ones :)

<img width="496" alt="Capture d’écran 2023-06-04 à 00 40 18" src="https://github.com/MadeInPierre/finalynx/assets/20896232/f58288c2-405e-42ff-8634-65a8359920cd">

[EDIT 04/06/2023: Updated portfolio scheme]

**Reply by MadeInPierre** [reply]
Awesome! Thanks for sharing, it's satisfying to see how other people use this project ^^ 

Pro tip just in case💡 in my personal config, I created a `display_mode` variable to set the expanded/collapsed mode of some of my folders all at the same time

```python
display_mode = FolderDisplay.COLLAPSED  # or EXPANDED or LINE

portfolio = Portfolio(children=[
    ...
        ...
        Folder(..., display=display_mode),
])
```

Then, my script starts up in COLLAPSED mode every morning, and I can set it to EXPANDED mode when I want to make micro decisions ^^

Customizable colors are coming soon 🙏
